internal/app/parcel/repository.go

```diff
@@ -0,0 +1,39 @@
+package parcel
+
+import (
+	"context"
+	"fmt"
+	"parcel-service/internal/app/model"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+)
+
+// SQL Query and error
+const (
+	errUniqueViolation = pq.ErrorCode("23505")
+	insertParcelQuery  = `INSERT INTO parcel (user_id, carrier_id, source_address, destination_address, source_time, type, price, carrier_fee, company_fee) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)`
```

Comment:
at the time of parcel creation carrier_id will be empty so remove carrier_id from query